docker-app/qfieldcloud/core/views/projects_views.py

```diff
@@ -77,6 +77,8 @@ class ProjectViewSet(viewsets.ModelViewSet):
     lookup_url_kwarg = "projectid"
     permission_classes = [permissions.IsAuthenticated, ProjectViewSetPermissions]
     pagination_class = pagination.QfcLimitOffsetPagination()
+    filter_backends = [filters.OrderingFilter]
+    ordering_fields = ["owner__username", "name", "created_at"]
```

Comment:
@suricactus am wondering about the `owner__username` here, does the "sort by owner" mean by username ?

Reply:
It does, correct!